### PR TITLE
Fix group PUT, GET by id, Fix form answers resource(Feature/#155,160)

### LINKED ISSUE #155: Change db
- remove Answers table
- change FormResults answers

### LINKED ISSUE #160: Fix group PUT, GET by id, Fix form answers
Save form answers directly in from results in format:
{"question": "answer",
 "question": answer, ...}

### PR DESCRIPTION
**Pull request by dtymkiv**
- Anonymous are able to pass forms now.
- Deleted Answers model and service **(NEW **MIGRATION!!!)**.
- All answers are now in form_result table.
- Fix group PUT, GET/{group_id} responses.

### CODE REVIEW

**Review by IvanKostiv — DISMISSED** [review]

File: src/app/services/form_result.py [hunk 1]
```diff
@@ -280,11 +280,29 @@ def validate_data(form_result):
...
+        :param form_id:
+        :param answers:
+        :return: answers dictionary
+        """
+        form_fields = FormFieldService.filter(form_id=form_id)
+        result = {}
+        for form_field in form_fields:
+            for answer in answers:
+                if answer['position'] == form_field.position:
+                    result[form_field.question] = answer['answer']
```
Comment (yurdosii) [inline src/app/services/form_result.py:297]: break in if ?

File: src/app/routers/form_answer.py [hunk 2]
```diff
@@ -92,24 +84,25 @@ def post(self, form_id):
...
+        if not current_user.is_anonymous:
+            result['user_id'] = current_user.id
+        else:
+            result['user_id'] = None
+        result['answers'] = FormResultService.create_answers_dict(form_id, result['answers'])
         result = FormResultService.create(**result)
         if result is None:
             raise BadRequest("Cannot create result instance")
-        return Response(status=201)
+        result_json = FormResultService.to_json(result, many=False)
```
Comment (yurdosii) [inline src/app/routers/form_answer.py:102]: new line

File: src/app/models/form_result.py [hunk 3]
```diff
@@ -62,7 +62,19 @@ class Meta:
...
+    """
+    class Meta:
+        """
+        Schema meta
+        """
+        fields = ("id", "user_id", "form_id", "created", "answers")
+
+    answers = fields.Dict(required=True, keys=fields.String(), values=fields.Raw())
+    user_id = fields.String(data_key="userId")
+    form_id = fields.String(data_key="formId")
```
Comment (Howrran) [inline src/app/models/form_result.py]: id should be Integer

File: src/app/services/form_result.py [hunk 4]
```diff
@@ -280,11 +280,30 @@ def validate_data(form_result):
...
+        return result
+
     @staticmethod
     def validate_schema(data):
         """
         Validate data for FormResultSchema
         """
-        schema = FormResultSchema()
+        schema = FormResultPostSchema()
+        print(data)
```
Comment (yurdosii) [inline src/app/services/form_result.py]: Remove print

**Review by zzeropilot — DISMISSED** [review]

**Review by Howrran — DISMISSED** [review]

**Review by AndriiKret — DISMISSED** [review]

**Review by Howrran — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Review by yurdosii — APPROVED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Pull request merged**